environment/src/test/java/com/sequenceiq/environment/environment/validation/EnvironmentValidatorServiceTest.java

```diff
@@ -69,9 +76,15 @@
     @Mock
     private TagValidator tagValidator;
 
+    @Mock
+    private EntitlementService entitlementService;
+
     @InjectMocks
     private EnvironmentValidatorService underTest;
 
+    @InjectMocks
+    private EncryptionKeyUrlValidator encryptionKeyUrlValidator;
+
     @BeforeEach
     public void initTests() {
```

Comment:
I know this is not your change, but the usual practice is to call this `setUp()`. Could you please rename?